src/main/scala/com/lightbend/kafkalagexporter/AppConfig.scala

```diff
@@ -18,6 +18,11 @@ import scala.util.Try
 object AppConfig {
   def apply(config: Config): AppConfig = {
     val c = config.getConfig("kafka-lag-exporter")
+    val graphiteConfig: Option[GraphiteConfig] = (
+      for (host <- Try(c.getString("graphite-host"));
+           port <- Try(c.getInt("graphite-port")),
+             ) yield GraphiteConfig(
+               host, port, Try(c.getString("graphite-prefix")).toOption)).toOption
     val pollInterval = c.getDuration("poll-interval").toScala
     val lookupTableSize = c.getInt("lookup-table-size")
     val port = c.getInt("port")
```

Comment:
Support getting the prometheus port from here, and from the new report configuration section I mentioned above.  Use `kafka-lag-exporter.port` as the default and override with `kafka-lag-exporter.prometheus.port` if it exists.  This will maintain backwards compatibility.

Please update the `reference.conf` to include the `kafka-lag-exporter.reporters.prometheus` section.